build.sh

```diff
@@ -3,10 +3,14 @@
 set -o errexit
 [[ -n $DEBUG ]] && set -o xtrace
 
-declare -r ARTIFACT_DIRS="$(find cmd/* functional/* -maxdepth 0 -type d 2>/dev/null)"
-declare -r RELEASE_DIRS="$(find cmd/* -maxdepth 0 -type d 2>/dev/null)"
-declare -r SOURCE_PKGS="$(go list ./cmd/... ./pkg/... 2>/dev/null)"
-declare -r RELEASE_FILES="LICENSE README.md conf"
+ARTIFACT_DIRS="$(find cmd/* functional/* -maxdepth 0 -type d 2>/dev/null)"
```

Comment:
Per https://github.com/spiffe/spiffe/blob/master/CONTRIBUTING.md we try to follow google's shell style guide, https://google.github.io/styleguide/shell.xml?showone=Read-only_Variables#Read-only_Variables

Reply:
Oh and i should add that passing shellcheck is also a fine idea, just need to update the CONTRIBUTING doc

Reply:
The google shell style guide doesn't seem to require they are declared and assigned at the same time, in fact the example set the variable readonly after assignment *and* error checking.

Are you asking to move back to the single line version?

Reply:
Oh linked to the wrong part of the guide, this is the right link https://google.github.io/styleguide/shell.xml?showone=Constants_and_Environment_Variable_Names#Constants_and_Environment_Variable_Names

And you're right, the guide really isn't consistent. Comment retracted! shellcheck FTW!